lib/_http_server.js

```diff
@@ -295,6 +295,20 @@ ServerResponse.prototype.writeProcessing = function writeProcessing(cb) {
   this._writeRaw('HTTP/1.1 102 Processing\r\n\r\n', 'ascii', cb);
 };
 
+ServerResponse.prototype.writeEarlyHints = function writeEarlyHints(links, cb) {
+  let head = 'HTTP/1.1 103 Early Hints\r\n';
+
+  if (typeof links === 'string') {
```

Comment:
Perhaps we should be doing some kind of validation before blindly appending user input in this function?

Reply:
good idea, I added a validator based on a regex pattern > https://github.com/nodejs/node/pull/44180/files#diff-84047e9de2de28fd3ae796466cebdbae912d724504cb1b44c4f45b3517c1fa5fR261-R271